hoverxref/_static/js/hoverxref.js_t

```diff
@@ -143,7 +156,11 @@ $(document).ready(function() {
                 helper
             );
         }
-    })
+        {# If we're lazy-tipstering, we've used up the click event, so open the modal too #}
+    }){% if hoverxref_tooltip_lazy %}.tooltipster('open');
+    }); {# close lazy .one() wrapper #}
```

Comment:
It took me a while to understand the flow of these Jinja ifs. Would it be more readable/clear if we move this code to a function that we can call from both parts of the if clause?